    @Test
    public void dontRepeatErrorClaims() throws Exception {
        final FkProject project = new FkProject();
        new ClaimOut().type("Error").postTo(project);
        final int before = new LengthOf(new Claims(project).iterate())
            .intValue();
        new StkSafe(
            "errors1",
            new StkSafeTest.NonTestingFarm(),
            new StkSafeTest.StkError()
        ).process(
            project,
            new Claims(project).iterate().iterator().next()
        );
        MatcherAssert.assertThat(
            new Claims(project).iterate(),
            Matchers.iterableWithSize(before)
        );
    }